src/WebJobs.Script.Grpc/Server/AspNetCoreGrpcServer.cs

```diff
@@ -60,10 +60,11 @@ protected async ValueTask DisposeAsync(bool disposing)
         {
             if (!_disposed)
             {
-                if (disposing)
+                var temp = _grpcHost;
+                if (disposing && temp != null)
                 {
-                    await _grpcHost.StopAsync();
-                    _grpcHost.Dispose();
+                    await temp.StopAsync();
+                    temp.Dispose();
```

Comment:
cc @brettsam - I _think_ this is an existing issue, if we register (but never start) the server then dispose it, we'll null ref. I noticed in this PR (I'm not sure how it worked before this, some other delta in the testing landscape), but the fix I think is probably one to lift to dev independently - thoughts?

Reply:
yeah, good call -- can we move this to a separate PR and get it in?

Reply:
Done: #7981